packages/core/admin/admin/src/pages/AuthPage/components/Register/index.js

```diff
@@ -1,173 +1,228 @@
-                type="email"
-                validations={{ required: true }}
-                value={get(modifiedData, `${inputsPrefix}email`, '')}
-              />
-              <Input
-                error={formErrors[`${inputsPrefix}password`]}
-                label="Auth.form.password.label"
-                name={`${inputsPrefix}password`}
-                onChange={onChange}
-                type="password"
-                validations={{ required: true }}
-                value={get(modifiedData, `${inputsPrefix}password`, '')}
-              />
-              <Input
-                error={formErrors[`${inputsPrefix}confirmPassword`]}
-                label="Auth.form.confirmPassword.label"
-                name={`${inputsPrefix}confirmPassword`}
-                onChange={onChange}
-                type="password"
-                validations={{ required: true }}
-                value={get(modifiedData, `${inputsPrefix}confirmPassword`, '')}
-              />
-              <Flex alignItems="flex-start">
-                <Checkbox
-                  name={`${inputsPrefix}news`}
-                  onChange={onChange}
-                  value={get(modifiedData, `${inputsPrefix}news`, false)}
-                />
-                <Padded left size="sm" />
-                <CustomLabel
-                  id="Auth.form.register.news.label"
-                  values={{ terms, policy }}
-                  onClick={() => {
-                    onChange({
-                      target: {
-                        name: `${inputsPrefix}news`,
-                        value: !get(modifiedData, `${inputsPrefix}news`, false),
-                      },
-                    });
-                  }}
-                />
-              </Flex>
-              <Padded top size="md">
-                <Button type="submit" color="primary" textTransform="uppercase">
-                  {formatMessage({ id: 'Auth.form.button.register' })}
-                </Button>
-              </Padded>
-            </form>
+    <UnauthenticatedLayout>
+      <LayoutContent>
+        <Formik
+          enableReinitialize
+          initialValues={{
+            // TODO: Fix the initial data during the clean of the AuthLogin codebase
+            firstname: modifiedData?.userInfo?.firstname || '',
+            lastname: modifiedData?.userInfo?.lastname || '',
+            email: modifiedData?.userInfo?.email || '',
+            password: modifiedData?.userInfo?.password || '',
+            confirmPassword: modifiedData?.userInfo?.confirmPassword || '',
+            news: false,
+          }}
+          onSubmit={onSubmit}
+          validationSchema={schema}
+          validateOnChange={false}
+        >
+          {({ values, errors, handleChange }) => (
+            <Form noValidate>
+              <Main labelledBy="welcome">
+                <Column>
+                  <img src={authLogo} alt="" aria-hidden style={{ height: '72px' }} />
```

Comment:
I think you've used a "logo" component in the auth login no? If you don't use the dcomponent, could you please make the inline style a SC? 🙏🏻 😊